mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-linking-records.feature

```diff
@@ -0,0 +1,104 @@
+Feature: linking-rules tests
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json'  }
+    * def samplePath = 'classpath:spitfire/mod-entities-links/features/samples'
+
+    * def authorityNaturalId = karate.properties['authorityNaturalId']
+    * def authorityId = karate.properties['linkedAuthorityId']
+    * def instanceId = karate.properties['instanceId']
+    * def bibId = karate.properties['bibId']
+
+  @Positive
+  Scenario: Update linking authority 100 - should update bib and instance record content
+    # retrieve quick marc record
+    Given path '/records-editor/records'
+    And param externalId = authorityId
+    When method GET
+    Then status 200
+
+    # replace new field
+    * def record = response
+    * set record.relatedRecordVersion = 1
+
+    * def field = karate.jsonPath(record, "$.fields[?(@.tag=='100')]")[0]
+    * set field.content = '$a Updated'
+    * remove record.fields[?(@.tag=='100')]
+    * record.fields.push(field)
+
+    # update authority record
+    Given path '/records-editor/records', record.parsedRecordId
+    And request record
+    When method PUT
+    Then status 202
+
+    # retrieve bib record
+    Given path '/source-storage/records', bibId
+    When method GET
+    Then status 200
+    And match response.parsedRecord.content.fields[*].100.subfields[*].a contains only 'Updated'
+
+    # retrieve instance record
+    Given path '/instance-storage/instances', instanceId
+    When method GET
+    Then status 200
+    And match response.parsedRecord.content.fields[*].100.subfields[*].a contains only 'Updated'
```

Comment:
The instance doesn't have subfields

Reply:
How does it's even passed? Is this condition workable, do they really check smth?